utreexo-accumulator-bip.md

```diff
@@ -62,15 +108,8 @@ While a forest with 7 leaves will
 ```
 
 
-For example, an accumulator state with N=3 will look like so:
-
-
-It's possible to have less than 2^N elements in the accumulator. For an accumulator state where
-N=3 but with 6 elements will look like so:
-
-
-When adding another leaf to the accumulator when it's already allocated 2^N leaves will result in
-the accumulator resizing to hold 2^(N+1) leaves. For example, when adding a leaf to the accumulator
+When adding another leaf to the accumulator when it's already allocated $2^N$ leaves will result in
+the accumulator resizing to hold $2^(N+1)$ leaves. For example, when adding a leaf to the accumulator
```

Comment:
This renders weird. Only the first parenthesis is affected. This works ok for me:

$2^{N+1}$